advanced/src/resolvers/Query.test.js

```diff
@@ -0,0 +1,52 @@
+import test from "ava";
+import { request, GraphQLClient } from "graphql-request";
+
+/**
+  * The user token obtained by calling login mutation
+  * @see database/seed.graphql
+  * @see src/resolvers/mutation/auth
+  */
+const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJjamRnNGgwN3QwMDVkMDE3OW5yNDJzeTh4IiwiaWF0IjoxNTE4MTkyMzgwfQ.AsQb9P0BjWT_SOk4JSs7AYXrDr2VbRTdI3ExMItl-So"
```

Comment:
You need to dynamically generate this token as part of the test, because a JWT has an expiry date built-in.
This means that while the test might work now, it won't work at some point, when the JWT has expired.

Reply:
Yeah I wrote it in my notes :) I wasn't sure of the best approach though. Im thinking of calling the login mutation first and getting the token?